frontend/fleet/helpers.ts

```diff
@@ -344,6 +344,14 @@ export const humanHostDetailUpdated = (detailUpdated: string): string => {
   return moment(detailUpdated).fromNow();
 };
 
+export const hostTeamName = (team: string): string => {
+  if (!team) {
```

Comment:
Since this function receives a string, it would be best to check if the string is empty.